### PR TITLE
Add conda build package for iceFUNprog

### PR DESCRIPTION
**Pull request by y2kbugger**
see https://github.com/timvideos/litex-buildenv/pull/186

### CODE REVIEW

File: icefunprog/iceprog-only.patch [hunk 1]
```diff
@@ -0,0 +1,10 @@
+diff --git a/Makefile b/Makefile
```
Comment (mithro) [inline icefunprog/iceprog-only.patch]: Remove this file

File: icefunprog/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG or 'v0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: icefunprog
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/y2kbugger/iceFUNprog.git
+  git_rev: makefile
```
Comment (mithro) [inline icefunprog/meta.yaml]: Should this be master now?

Comment (y2kbugger) [reply icefunprog/meta.yaml]: fixed in https://github.com/litex-hub/litex-conda/pull/79/commits/7bdab676a07d14eefe0ff80f852cf1ceea31cfaf

File: icefunprog/iceprog-only.patch (hunk 1 above)
Comment (y2kbugger) [reply icefunprog/iceprog-only.patch]: fixed in https://github.com/litex-hub/litex-conda/pull/79/commits/c6075187c67809af86c8f69274e72ded742b38c1

**Pull request merged**